### PR TITLE
Let bug

### PR DESCRIPTION
**Pull request by aboyton**
This is the bug I found when doing #189 but pulled into separate pull request.

I have no idea why this pull request is taking in other commits of yours. I've tried doing it several times but I'm obviously doing something wrong. The commit itself should just be in https://github.com/aboyton/lebab/commit/8466102552e8d96c3611260c3247c3541bde57df.

### CODE REVIEW

File: test/transform/letTest.js [hunk 1]
```diff
@@ -460,6 +460,15 @@ describe('Let/const', () => {
       );
     });
 
+    it('should ignore variable declared inside a block but modified outside', () => {
```
Comment (nene) [inline test/transform/letTest.js]: It's not actually a modification. Variable `x` is not modified. Rather the point is that the variable is used in computed property access.

File: src/utils/variableType.js [hunk 2]
```diff
@@ -47,7 +47,8 @@ function isDeclaredVariable(node, parent) {
 }
 
 function isPropertyInMemberExpression(node, parent) {
-  return parent.type === 'MemberExpression' && parent.property === node;
+  return parent.type === 'MemberExpression' && parent.property === node && !parent.computed;
+  // return parent.type === 'MemberExpression' && parent.property === node;
```
Comment (nene) [inline src/utils/variableType.js]: Commented out code?

**Comment by nene** [comment]
Looks like you have rebased something along the way. You are missing one merge-commit right before the 2.5.0 commit that exists in lebab repo. This merge could have been a fast-forward merge, but I forced an explicit merge, might be something to do with this.

I can just cherry-pick your single commit, though it contains some commented-out code and the test description is a bit off.

**Comment by nene** [comment]
Cherry-picked the change and corrected the minor flaws.

**Comment by aboyton** [comment]
Thanks for cherry picking and fixing the minor things.

**Pull request closed without merging**